.tf_configure.bazelrc

```diff
@@ -0,0 +1,16 @@
+build --action_env PYTHON_BIN_PATH="/usr/local/bin/python3.9"
```

Comment:
I think this and `tools/python_bin_path.sh` should be removed. (I'll add `.tf_configure.bazelrc` to the gitignore, I'm surprised this hasn't happened before)

Reply:
removed both. Sorry for the mess

Reply:
No worries at all!!